### PR TITLE
FIX: fixed some css added some css

### PR DESCRIPTION
**Pull request by deepansh96**

### CODE REVIEW

File: src/components/PlioQuestion.vue [hunk 1]
```diff
@@ -16,8 +16,8 @@
               id="skip-button"
               @click="clickSkip"
             >
-              <div class="leftright"></div>
-              <div class="rightleft"></div>
+              <i class="fas fa-window-close" 
+                 style="font-size: 1.3em; color: crimson;"></i>
```
Comment (pritamps) [inline src/components/PlioQuestion.vue:20]: Add style to CSS, not in template

File: src/components/PlioQuestion.vue [hunk 2]
```diff
@@ -443,47 +451,68 @@ input {
...
     text-align: center;
     text-decoration: none;
     margin: 4px 2px;
     transition-duration: 0.4s;
     cursor: pointer;
+    height: 2.5em;
+    font-size: 1.4rem;
   }
 
   .submit {
```
Comment (pritamps) [inline src/components/PlioQuestion.vue:469]: Just a note. start learning SCSS. CSS will look much cleaner!

**Review by pritamps — CHANGES_REQUESTED** [review]

File: src/components/PlioQuestion.vue (hunk 2 above)
Comment (deepansh96) [reply src/components/PlioQuestion.vue:469]: I thought SCSS can't run in all/old/stupid browsers. 
SCSS needs to be converted into normal CSS right? And vue does that automatically or are we using some package for that?

File: src/components/PlioQuestion.vue (hunk 1 above)
Comment (deepansh96) [reply src/components/PlioQuestion.vue:20]: Whoops, sorry forgot!

File: src/components/PlioQuestion.vue (hunk 2 above)
Comment (pritamps) [reply src/components/PlioQuestion.vue:473]: Yes yes, Vue will do it automatiaclly. Or we'll need a package (that won't be included in the build -- will only run while building)

I don't think any browser can read SCSS. 

ANyway, hugo ka website ka CSS SCSS mein hi hai. Seekh hi loge :D

**Review by pritamps — APPROVED** [review]

**Pull request merged**